homeassistant/components/webostv/manifest.json

```diff
@@ -3,8 +3,9 @@
   "name": "Webostv",
   "documentation": "https://www.home-assistant.io/integrations/webostv",
   "requirements": [
-    "pylgtv==0.1.9",
-    "websockets==6.0"
+    "aiopylgtv==0.2.0",
+    "websockets==8.0.2",
+    "numpy==1.17.4"
```

Comment:
Why do we need numpy?

Reply:
aiopylgtv also supports the calibration functions of these TV's now, which involves some look-up-tables which are implemented with numpy.  But indeed it depends on numpy so we don't need to add this requirement explicitly.